### DISCUSSION TITLE
建议支持IP所属地显示

### DISCUSSION THREAD

**Discussion by Misaka-Mikoto-Tech** [the1812/Bilibili-Evolved#3522] [功能建议 / Ideas]
https://greasyfork.org/zh-CN/scripts/448434-b%E7%AB%99%E8%AF%84%E8%AE%BA%E5%8C%BA%E5%BC%80%E7%9B%92
B站现在的IP所属地只有app端显示，现在有个脚本可以显示评论区的实时IP，但无法显示用户首页的常用IP所在地，Bilibili-Evolved能否一步到位的支持呢

**Comment by the1812** [comment]
我觉得官方很快也要上了，之前评论区改版的时候就多了个显示地区的空容器

**Reply by ChaXxl** [reply]
哈哈哈，一年了，啊 B 还没做呢，我看啊 B 就是不想做，嫌麻烦 🤭

**Reply by the1812** [reply]
(一年了各种地方皮肤换了不少

**Comment by Geeyun-JY3** [comment]
我感觉 Bilibili Evolved 一直都是倾向于官方能做出来就不搞的（

而且你先别急，这事儿破站肯定比你更急（

**Reply by AnzhiZhang** [reply]
真的吗

**Comment by loktindyi** [comment]
App都上了，网页还会远吗

**Reply by AnzhiZhang** [reply]
真的吗
